src/Sylius/Bundle/UserBundle/Form/Type/CustomerSimpleRegistrationType.php

```diff
@@ -51,6 +51,8 @@ public function buildForm(FormBuilderInterface $builder, array $options = [])
             ])
             ->add('user', 'sylius_user_registration', [
                 'label' => false,
+                'validation_groups' => $this->validationGroups,
```

Comment:
We inject validation groups to `UserRegistrationType`, so we shouldn't need to do this.